addons/payment_demo/tests/test_payment_transaction.py

```diff
@@ -80,3 +80,62 @@ def test_making_a_payment_request_propagates_token_simulated_state_to_transactio
             tx.token_id = self._create_token(demo_simulated_state=state)
             tx._send_payment_request()
             self.assertEqual(tx.state, tx.token_id.demo_simulated_state)
+
+    def test_get_tx_from_notification_data_returns_partial_capture_child_tx(self):
+        """ Test that get_tx_from_notification_data returns child transaction if the capture is partial. """
```

Comment:
nitpicking: a line should be max 100 chars long

Reply:
Additionally: since it's almost the name of this test, is this docstring needed? :thinking: